script.js

```diff
@@ -243,3 +243,49 @@ ${dynamicImage}
     }
   });
 }
+
+// Javascript for the local storage
+const userDataInLocalStorage = localStorage.getItem('userData');
+const parsedData = JSON.parse(userDataInLocalStorage);
+document.getElementById('fullName').value = parsedData.name;
+document.getElementById('email').value = parsedData.email;
+document.getElementById('textArea').value = parsedData.comment;
+function localStorageMine() {
+  const userName = document.getElementById('fullName');
+  const userEmail = document.getElementById('email');
+  const userComment = document.getElementById('textArea');
+  const uname = userName.value;
+  const email = userEmail.value;
+  const comment = userComment.value;
+  const inputFieldObject = {
+    name: uname,
+    email,
+    comment,
+  };
```

Comment:
- [ ] Well done 👏🏻 , you did an awesome job by implementing the changes as per the previous code reviewer's request 👍🏻 but you guys still not following the things properly, you need to work on it before your project could be approved. It is required that when the user loads the page, the code checks if there are any saved data in the local storage and if so the input fields are pre-filled with the data. What this means is that, if the user enters data into the form without submitting the form, when the user reloads the page, the user's data should remain on the form. Kindly check the below code for insight 👇

```javascript
    window.onload = () => {
      let savedFormData = localStorage.getItem('contactFormInput');
      savedFormData = JSON.parse(savedFormData);
      // Check if the form data object is found on localStorage
      if (savedFormData) {
        // populate inputs values if data was found
        // ex: nameInput.value = savedFormData.name
       }
    };

```

### Video Demo For Understanding
[Screencast from 01-07-23 09:24:27 AM IST.webm](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/66476329/0e3904ac-fbf7-4ad5-9e1d-a422455d9567)

### Please resolve the below console error
![Screenshot from 2023-07-01 09-22-05](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/66476329/7e438c33-3ced-4292-b3f3-adf7a4ad4a6e)